src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -150,6 +151,7 @@ private void CheckForShutdown()
 
                     if (_clientControl != null)
                     {
+                        await _sendSettingsTask.ConfigureAwait(false);
```

Comment:
Is it possible this task throws? E.g. connection closed before we were able to send settings? Or it is impossible at this point in time?

Reply:
No, `SendSettingsAsync` catches all exceptions and calls `Abort` on them:
https://github.com/dotnet/runtime/blob/f69470a91ac823962f3dcb919f8a92f4bb85bde1/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs#L389-L392